### DISCUSSION TITLE
Version 5.7.1 is out

### DISCUSSION THREAD

**Discussion by bdergin** [rocm-arch/rocm-arch#974] [General]
https://github.com/RadeonOpenCompute/ROCm  <--- new version has been out for 2 months, just wondering why the arch version wasn't updated.

**Comment by acxz** [comment]
The pkgbuilds here get updated incrementally, as in there might be an issue with one pkgbuild that is preventing us to upgrade that particular package to 5.7.1 but another package is working just fine for us to upgrade it. You'll have to either be patient as the packages get slowly updated as the community has time to update them/submit PRs to do so or you can help an individual package along by attempting to build the new version and sort out any kinks/report issues with it.
